package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "11.0.1",
+  "version": "14.0.0",
```

Comment:
Why is this going from 11 straight to 14?

Reply:
The master branch is on `13.x.x` right now. No one has updated the `package-lock.json` since `11.0.1`, so that's why it's going from 11 to 13.

Reply:
@f1337 is there a way to update the GH bot to raise an error when `package.json` version is updated but `package-lock.json` is not? That may help keep them in sync.

Reply:
For now we will just remove `package-lock.json` and ignore it from the repo in order to make it clear that the yarn lockfile is the one that gets used.

Reply:
I've removed the package-lock.json version change.